charts/engine/templates/engine-job-template-gpu-cm.yaml

```diff
@@ -37,7 +39,15 @@ data:
 {{ if .Values.schedulerName }}
           schedulerName: {{ .Values.schedulerName }}
 {{ end }}
+{{ if .Values.addGpuTolerations }}
+          tolerations:
+            - key: nvidia.com/gpu
+              operator: Exists
+              effect: NoSchedule
+{{ end}}
```

Comment:
I don't think we should add this

Reply:
Might be, @yotamazriel and I search what are the changes between running jobs manifests(alongside `run-ai` agent) with not running jobs manifests.
The toleration was one of the differences